<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>776</width>
    <height>617</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>776</width>
    <height>617</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SHAFT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Source.qrc">
    <normaloff>:/Source/0_f.bmp</normaloff>:/Source/0_f.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_STAGE">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>STAGE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TIME">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>230</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TIME:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_time">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>231</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Chara_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>261</width>
      <height>81</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>261</width>
      <height>81</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/chara.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bg_side">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>-120</y>
      <width>291</width>
      <height>751</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>751</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>751</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/main_bg.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Chara_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>261</width>
      <height>81</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>261</width>
      <height>81</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/chara.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="HP_Bar1">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>25</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/hp_bar.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="HP_Bar2">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>104</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/hp_bar.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bg_black">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>520</width>
      <height>617</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>520</width>
      <height>617</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>520</width>
      <height>617</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="top_spike1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/top_spike.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="top_spike1_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/top_spike.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="top_spike1_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/top_spike.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_stage">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="P1_s">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>22</y>
      <width>40</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="P2_s">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>102</y>
      <width>40</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_RECORD">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>RECORD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_P1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>P1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_P2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>P2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pause">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>285</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PAUSE: P</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Gameover">
    <property name="geometry">
     <rect>
      <x>999</x>
      <y>999</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>520</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>520</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>  GAME OVER (space to continue)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pause_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>325</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>MUTE : M</string>
    </property>
   </widget>
   <widget class="QLabel" name="MP_Bar1">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>53</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/mp_bar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MP_Bar2">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>133</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Source.qrc">:/Source/mp_bar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>bg_black</zorder>
   <zorder>top_spike1_3</zorder>
   <zorder>bg_side</zorder>
   <zorder>label_STAGE</zorder>
   <zorder>label_TIME</zorder>
   <zorder>lcdNumber_time</zorder>
   <zorder>Chara_1</zorder>
   <zorder>Chara_2</zorder>
   <zorder>HP_Bar1</zorder>
   <zorder>HP_Bar2</zorder>
   <zorder>top_spike1</zorder>
   <zorder>top_spike1_2</zorder>
   <zorder>lcdNumber_stage</zorder>
   <zorder>P1_s</zorder>
   <zorder>P2_s</zorder>
   <zorder>label_RECORD</zorder>
   <zorder>label_P1</zorder>
   <zorder>label_P2</zorder>
   <zorder>label_Pause</zorder>
   <zorder>label_Gameover</zorder>
   <zorder>label_Pause_2</zorder>
   <zorder>MP_Bar1</zorder>
   <zorder>MP_Bar2</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Source.qrc"/>
 </resources>
 <connections/>
</ui>
